README.md

```diff
@@ -17,18 +17,17 @@ limitations under the License.
 ![](https://github.com/huggingface/optimum-habana/blob/main/readme_logo.png)
 
 
-# Optimum Habana
+# Optimum for Intel(R) Gaudi(R) Accelerator 
```

Comment:
@regisss  do you think the R can be changed to trade mark version?

Reply:
Can you try `:registered:`? This should give :registered:
Or are you talking about :tm:?

Reply:
do I need ( ) around it?

Reply:
I don't think so

Reply:
seems not working, I have to copy it somehow